Services/NewsService/Properties/launchSettings.json

```diff
@@ -10,7 +10,7 @@
   "profiles": {
     "IIS Express": {
       "commandName": "IISExpress",
-      "launchBrowser": true,
+      "launchUrl": "https://localhost:44384/swagger",
```

Comment:
What is this line for ? What swagger do here?

Reply:
swagger was deleted